contrib/nitiwiki/src/wiki_base.nit

```diff
@@ -560,6 +583,28 @@ class WikiArticle
 	redef fun to_s do return "{name} ({parent or else "null"})"
 end
 
+# The sidebar is displayed in front of the main panel of a `WikiEntry`.
+class WikiSidebar
+
+	# Wiki used to parse sidebar blocks.
+	var wiki: Nitiwiki is lazy do return entry.wiki
```

Comment:
what is the point of a `var` here? a `fun` would be enough as there is no real need to store more state than the related entry.